tap_slack/transform.py

```diff
@@ -33,9 +33,10 @@ def transform_json(stream, data, date_fields, channel_id=None):
             for date_field in date_fields:
                 timestamp = record.get(date_field, None)
                 if timestamp and isinstance(timestamp, str):
-                    if stream == 'messages' or stream == "threads" and date_field == 'ts':
+                    if stream == "messages":
                         record['thread_ts'] = timestamp
-                        record[date_field] = decimal_timestamp_to_utc_timestamp(timestamp)
-                    else:
-                        record[date_field] = decimal_timestamp_to_utc_timestamp(timestamp)
+                    elif stream == 'threads' and date_field == 'ts':
+                        if 'thread_ts' in record:
+                            record['thread_ts'] = record.get['thread_ts']
```

Comment:
1. what is the purpose of setting the value to itself? 
2. the syntax is incorrect. Please write a test with mocked HTTP request to validate that this works correctly.